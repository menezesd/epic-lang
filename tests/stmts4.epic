func helper(a,b) {
    // note that "else" must associate with the nearest "if" here.
    if a then if b then print 1; else print 2;
    print none;
}

func main() {
    for i in 0..2 do {
        for j in 0..2 do {
            x = i==1;
            y = j==1;
            helper(x, y);
        }
    }
}