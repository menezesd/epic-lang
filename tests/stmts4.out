None
None
2
None
1
None
